test/e2e/discovery/service_discovery.go

```diff
@@ -238,47 +251,79 @@ func RunServicesPodAvailabilityMutliClusterTest(f *lhframework.Framework) {
 		f.AwaitServiceImportIP(framework.ClusterA, nginxServiceClusterA)
 	}
 
-	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterA, netshootPodList, checkedDomains, true)
-	verifyServiceIpWithDig(f.Framework, framework.ClusterA, nginxServiceClusterB, netshootPodList, checkedDomains, true)
+	clusterBDomain := getClusterDomain(f.Framework, framework.ClusterB, netshootPodListB)
```

Comment:
What's the functional purpose of `clusterBDomain`? It's only used for output on L256.

Reply:
Removed